tests/conftest.py

```diff
@@ -2242,3 +2250,38 @@ def mock_config(rnafusion_dir: Path, rnafusion_case_id: str) -> None:
     Path(rnafusion_dir, rnafusion_case_id, f"{rnafusion_case_id}_samplesheet.csv").touch(
         exist_ok=True
     )
+
+
+@pytest.fixture
+def store_with_sequencing_metrics(store: Store):
+    """Return a store with multiple samples with sample lane sequencing metrics."""
+
+    sample_sequencing_metrics_details = [
```

Comment:
```suggestion
    sample_sequencing_metrics_row: List[Union[str, int, float]] = [
```

Reply:
This is an instance where I think the type hint does not add value and makes the code more difficult to read. But I will add it since our verbal coding standard says we always should add type hints no matter what.

```python
sample_sequencing_metrics_details: List[Union[str, str, int, int, float, int]]
```